spec/features/manage_proposals/approver_workflow_spec.rb

```diff
@@ -8,6 +8,50 @@
   end
 
   context 'when loading the manage proposals page' do
+    context 'when publishing a create metadata proposal' do
+      let(:concept_id)                { cmr_client.get_collections_by_post({'EntryTitle':'Date-Check','ShortName': 'Proposal for checking MetadataDates'}, nil, 'umm_json').body.dig('items',0,'meta','concept-id') }
+      let(:concept)                   { cmr_client.get_concept(concept_id, 'access_token', {}).body }
+      let(:short_name)                { 'Proposal for checking MetadataDates' }
+      let(:create_date_after_publish) { Time.parse(concept.dig('MetadataDates',0,'Date')) }
+      let(:update_date_after_publish) { Time.parse(concept.dig('MetadataDates',1,'Date')) }
+
+      before do
+        # FactoryBot respects validations; need to be in proposal mode to create
+        set_as_proposal_mode_mmt(with_draft_approver_acl: true)
+        @native_id = 'dmmt_collection_2'
+        @proposal = create(:collection_draft_proposal_all_required_fields, proposal_short_name: short_name, proposal_entry_title: 'Date-Check', proposal_request_type: 'create', proposal_native_id: @native_id)
+        mock_approve(@proposal)
+        # Workflow is in mmt proper, so switch back
+        set_as_mmt_proper
+        # Mock URS communication because it has no concept of our test users
+        mock_valid_token_validation
+        visit manage_proposals_path
+
+        within '.open-draft-proposals tbody tr:nth-child(1)' do
+          click_on 'Publish'
+        end
+        select 'MMT_2', from: 'provider-publish-target'
+        within '#approver-proposal-modal' do
+          click_on 'Publish'
+        end
+      end
+
+      it 'displays expected success messages' do
+        expect(page).to have_content('Collection Metadata Successfully Published!')
+      end
+
+      it 'deletes the proposal in dMMT' do
+        expect(CollectionDraftProposal.where(short_name: short_name).count).to eq(0)
+      end
+
+      it 'contains the correct CREATE and UPDATE dates' do
```

Comment:
I think it is worth checking that the proposal did not have these dates before being published. Should trigger an alarm if somebody modifies the factory, for some reason.